src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-100100.mdx

```diff
@@ -0,0 +1,42 @@
+---
+subject: 'Python agent'
+releaseDate: '2024-10-10'
+version: 10.1.0
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Add support for Python 3.13', 'Add new Large LLM custom attribute context manager API', 'Add support for reporting of Amazon ECS Fargate Docker IDs', 'Add instrumentation for `SQLiteVec` vectorstore in Langchain']
```

Comment:
We should either say "Large Language Model" or "LLM" but "Large LLM" is redundant

Reply:
Oops 🤣